migrations/gratuit-heberge.js

```diff
@@ -0,0 +1,43 @@
+var mongoose = require('mongoose');
```

Comment:
To improve discoverability, use version numbers as file names: `v<origin>_v<destination>`.